client/src/js/todos.js

```diff
@@ -0,0 +1,52 @@
+import { $ } from './util';
+import { createCardTemplate, createColumnTemplate } from './template';
+
+const getData = async (keyword) => {
+  const url = `http://localhost:3001/${keyword}`;
+  const res = await fetch(url);
+  const json = await res.json();
+  return json;
+};
+
+export const renderColumns = async () => {
+  const columnData = await getData('columns');
+  columnData.forEach((title) => {
+    $('main').insertAdjacentHTML('beforeend', createColumnTemplate(title));
```

Comment:
forEach로 html을 추가하게 되면 반복문 하나하나마다 계속해서 html이 추가되게 됩니다. 
map을 사용하여 하나의 html 덩어리를 한번에 넣어주면 성능적으로 좋아질 것 같습니다.
```suggestion
    const columns = columnData.map(title => createColumnTemplate(title));
    $('main').insertAdjacentHTML('beforeend', columns.join());
```

Reply:
성능 차이가 있다는건 처음 알게 되었네요! 반영해서 수정하도록 하겠습니다!